### PR TITLE
Add tests for case sensitive database

### PR DESCRIPTION
**Pull request by lucyzhang929**
Adding three test cases:
1. query from case sensitive database
2. upsert to case sensitive database
3. upserting POCO with mismatched casing fails

### CODE REVIEW

File: test/Integration/test-js/AddProductIncorrectCasing/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/**
```
Comment (Charles-Gagnon) [inline test/Integration/test-js/AddProductIncorrectCasing/index.js:1]: copyright (could you follow up with adding linting for the test files too? Looks like it's just doing the samples folder currently)

File: test/Integration/test-powershell/AddProductIncorrectCasing/run.ps1 [hunk 2]
```diff
@@ -0,0 +1,18 @@
+using namespace System.Net
```
Comment (Charles-Gagnon) [inline test/Integration/test-powershell/AddProductIncorrectCasing/run.ps1:1]: I suppose these should also have a copyright header. @VasuBhog Can you make an issue to follow up adding that for the powershell files? (ideally with some sort of linting to ensure it's not missed in the future...not sure what exists out there for that)

**Review by Charles-Gagnon — APPROVED** [review]

File: test/Integration/test-js/AddProductIncorrectCasing/index.js (hunk 1 above)
Comment (lucyzhang929) [reply test/Integration/test-js/AddProductIncorrectCasing/index.js:4]: Will send a follow up PR for linting test-js folder

File: test/Integration/test-powershell/AddProductIncorrectCasing/run.ps1 (hunk 2 above)
Comment (lucyzhang929) [reply test/Integration/test-powershell/AddProductIncorrectCasing/run.ps1:1]: Created a follow up issue here: https://github.com/Azure/azure-functions-sql-extension/issues/610

**Review by MaddyDev — APPROVED** [review]

**Pull request merged**